packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.scss

```diff
@@ -1,317 +1,43 @@
 @import '../../common/common';
-@import '../../common/_focusBorder';
-
-// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
-
-//
-// Office UI Fabric
-// --------------------------------------------------
-// Command Bar styles
-//
-// Using Padding instead of Margin for ms-CommandBarItem so that width to
-// calculate overFlow items is calculated correctly.
-
-$CommandBar-height: 40px;
-$CommandBar-mobileHeight: 48px;
-$CommandBar-iconSize: 18px;
-$CommandBar-smallLeft: 44px;
-$CommandBar-largeLeft: 48px;
-$CommandBar-xLargeLeft: 232px;
-$CommandBar-sideAreaWidthsm: 128px;
-$CommandBar-sideAreaWidthlg: 260px;
-$CommandBar-sideAreaWidthxlg: 345px;
-
-$CommandBarItem-height: $CommandBar-height;
-$CommandBarItem-marginTop: 13px;
-$CommandBarItem-textHeight: 16px;
-$CommandBarItem-fontSize: 16px;
-$CommandBarItem-chevronDown-fontSize: 12px;
-$CommandBarItem-overflow-fontSize: 18px;
-$CommandBarItem-iconLabelMargin: 8px;
-$CommandBarItem-flippedLineHeight: 32px;
-
-$SearchBox-height: 40px;
-$SearchBox-widthLgCollapsed: 50px;
-$SearchBox-widthMdDownCollapsed: 36px;
-$SearchBox-widthXLarge: 208px;
-$SearchBox-iconSize: 16px;
-$SearchBox-sidePadding: 8px; // padding in input on left and right sides without icons
 
 .root {
   @include ms-font-m;
   color: $ms-color-neutralPrimary;
-
   position: relative;
+  display: flex;
   background-color: $ms-color-neutralLighter;
-  height: $CommandBar-height;
   white-space: nowrap;
   user-select: none;
-}
-
-.container {
-  display: flex;
-}
-
-.primaryCommands {
-  overflow: hidden;
+  .search {
```

Comment:
The baked in search feels broken. It results in unecessary bytes being pulled down for people that don't use a search box.